### DISCUSSION TITLE
Can't access navigation items on Github Web Page for Firefox Mobile

### DISCUSSION THREAD

**Discussion by Gustl22** [community/community#76888] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi there,

On Firefox Mobile I can't access the navigation items for the GitHub Page, (such as Pull Requests), from the main menu. This bug occurs since one or two weeks...

In contrast to chrome the items are flipped to the top.

![IMG_20231124_175457](https://github.com/community/community/assets/18537755/6dc2286f-c9aa-4910-b1d1-8143e4b15886)

Firefox Mobile:
120.0 (Build #2015986298), 039de7ef85+
GV: 120.0-20231116134553
AS: 120.0.1

On Android 12

Screenshot of Chrome:

![IMG_20231124_180807](https://github.com/community/community/assets/18537755/d7855fbf-465b-4627-8122-1d9afb4057a3)

**Comment by RangHo** [comment]
I can confirm that the same issue can be reproduced both on my mobile phone (Android 13, Firefox) and PC (Windows 10 and Linux, Firefox), given that the Firefox window in the PC version is narrow enough to trigger the responsive UI to kick in.

**Comment by laurent22** [comment]
Why was that changed anyway. It was so much better when we could horizontally slide the menu items.

Changing stuff for the sake of changing stuff and it doesn't even work

**Comment by yan12125** [comment, marked as answer]
Fixed in Firefox nightly as mentioned in https://github.com/orgs/community/discussions/75847
